google-cloud-bigquery/lib/google/cloud/bigquery/encryption.rb

```diff
@@ -0,0 +1,123 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "google/apis/bigquery_v2"
+
+module Google
+  module Cloud
+    module Bigquery
+      ##
+      # # Encryption Configuration
+      #
+      # A builder for BigQuery table encryption configurations, passed to block
+      # arguments to {Dataset#create_table} and
+      # {Table#encryption_configuration}.
+      #
+      # @see https://cloud.google.com/bigquery/docs/customer-managed-encryption
+      #   Protecting Data with Cloud KMS Keys
+      #
+      # @example
+      #   require "google/cloud/bigquery"
+      #
+      #   bigquery = Google::Cloud::Bigquery.new
+      #   dataset = bigquery.dataset "my_dataset"
+      #   encrypt_config = Google::Cloud::Bigquery::EncryptionConfiguration.new
+      #   encrypt_config.kms_key_name = "my_key_name"
+      #   table = dataset.create_table "my_table",
+      #                                encryption_configuration: encrypt_config
+      #
+      class EncryptionConfiguration
```

Comment:
This file should be named `encryption_configuration.rb`.

Reply:
Done. Renamed `encryption.rb` to `encryption_configuration.rb`.